Source/WebCore/style/StyleChangedAnimatableProperties.h

```diff
@@ -32,6 +32,10 @@ struct CSSPropertiesBitSet;
 
 namespace Style {
 
+// Conservatively answers what CSS properties we should visit for CSS transitions. We do not need
+// to precisely check equivalence before saying "this property needs to be visited". This is tuned
+// based on Speedometer3.0 data.
```

Comment:
```suggestion
// based on Speedometer 3.0 data.
```
I’m not sure the “is tuned” comment is quite right in terms of verb tense. Maybe more like “was tuned in 2025 based on Speedometer 3.0 data”?